README.md

```diff
@@ -24,6 +24,21 @@ The front end serves as a convinient way to see the responses served by the back
 
 The emulator is a simple HTTP server running on [Sinatra](https://sinatrarb.com/), a Ruby server library. 
 
+## Run on docker
```

Comment:
You will need to make sure this file is run through Prettier and Vale before merging

Reply:
Have linted the code and ran `Prettier` and `Vale`